packages/@aws-cdk/aws-codedeploy/test/lambda/deployment-group.test.ts

```diff
@@ -43,6 +43,10 @@ describe('CodeDeploy Lambda DeploymentGroup', () => {
           'Arn',
         ],
       },
+      AlarmConfiguration: {
+        Enabled: false,
+        Alarms: Match.absent(),
+      },
```

Comment:
Please add a test that confirms that `AlarmConfiguration` can still be enabled.

Reply:
Is it not already covered in existing tests?

https://github.com/cecheta/aws-cdk/blob/3c7e9b97515bbafa9919b69ba074b1313b02e389/packages/%40aws-cdk/aws-codedeploy/test/lambda/deployment-group.test.ts#L238